theano/gof/graph.py

```diff
@@ -117,6 +117,9 @@ def __init__(self, op, inputs, outputs):
                 if output.owner is None:
                     output.owner = self
                     output.index = i
+                    if not hasattr(output, "tag"):
+                        output.tag = utils.scratchpad()
```

Comment:
While thinking about this, I think Theano variable should always have a tag attribute. I would like to understand when they don't have it and fix the root cause if possible.

Do you know more about when this case happen? Unpickle of old graph?